linkcheck-local-skip-list

```diff
@@ -11,3 +11,4 @@ https:\/\/support.commercetools.com
 \/.+\/sdk
 \/.+\/tutorials
 \/.+\/getting-started
+\#ctp:.+:endpoint
```

Comment:
I had to disable linkcheck for all the links containing endpoint anchor because linkcheck seems to fail when checking those links: https://github.com/commercetools/commercetools-docs-kit/actions/runs/3574162285/jobs/6009092000
I assume it fails because the hash part of the link (which is url escaped) doesn't match exactly what contained in the anchor id (which is not url escaped)